test/_assets/samples/ckeditor-dev-correct/samples/old/api.html

```diff
@@ -202,8 +202,7 @@ <h1 class="samples">
 			CKEditor - The text editor for the Internet - <a class="samples" href="http://ckeditor.com/">http://ckeditor.com</a>
```

Comment:
Use HTTPS (fix in all samples):

```suggestion
			CKEditor - The text editor for the Internet - <a class="samples" href="https://ckeditor.com/">https://ckeditor.com/</a>
```